DevCycle.SDK.Server.Common/Model/Local/DVCLocalOptions.cs

```diff
@@ -12,20 +12,26 @@ public class DVCLocalOptions : IDVCOptions
         public bool DisableAutomaticEvents { get; set; }
         public bool DisableCustomEvents { get; set; }
         public int MaxEventsInQueue { get; set; }
+        public int FlushEventQueueSize { get; set; }
         public string EventsApiUri { get; set; }
 
         public string EventsApiSlug { get; set; }
         public Dictionary<string, string> CdnCustomHeaders { get; set; }
         public Dictionary<string, string> EventsApiCustomHeaders { get; set; }
 
-        public DVCLocalOptions(int configPollingIntervalMs = 1000, int configPollingTimeoutMs = 5000,
+        public DVCLocalOptions(
```

Comment:
Not sure if this is the public interface for all these options, but can we make sure all these options are here: https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/nodejs/src/types.ts#L58